tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
+	pylint3 --disable=I --reports=n ../templates/manage_certs.py
+	PYTHONPATH=../templates pylint3 --max-line-length=120 --disable=missing-docstring --reports=n test_manage_certs.py
+
+venv-unit-tests: unit_test_requirements.txt
+	virtualenv -p "`which python3`" "$@"
+	"$@"/bin/pip install -r "$<"
+
+venv-integration-tests: integration_test_requirements.txt
+	virtualenv "$@"
+	"$@"/bin/pip install -r "$<"
+
+unit_tests: venv-unit-tests
+	PYTHONPATH=../templates venv-unit-tests/bin/python -m unittest test_manage_certs.py
```

Comment:
@smarnach Could you also generate a coverage report? No need to make it fail below a given percentage for now, but it would help to see what is being run by the unit tests.